src/Raven.Studio/typescript/viewmodels/database/tasks/editRavenEtlTask.ts

```diff
@@ -277,6 +277,10 @@ class editRavenEtlTask extends viewModelBase {
     }
 
     private initObservables() {
+        const model = this.editedRavenEtl();
+        
+        this.showAdvancedOptions(!!model.loadRequestTimeout());
```

Comment:
please move that logic to view. We talked about that: models can have method - it is is great please to put logic instead of spoil that around. 

If you name this method: hasAnyAdvancedOption defined, it is easier in future to keep track of that if you add new field.

Reply:
done